<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BehaviourSettingsPage</class>
 <widget class="QWidget" name="BehaviourSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>851</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Behavior</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="b_show_tray_icon_">
     <property name="text">
      <string>Show tray icon</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="b_scroll_tray_icon_">
     <property name="text">
      <string>Scroll over icon to change track</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="b_keep_running_">
     <property name="text">
      <string>Keep running in the background when the window is closed</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="b_warn_close_playlist_">
     <property name="text">
      <string>Warn me when closing a playlist tab</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Language</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_13">
      <item>
       <widget class="QComboBox" name="language">
        <item>
         <property name="text">
          <string>Use the system default</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>You will need to restart Clementine if you change the language.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="startup_group_">
     <property name="title">
      <string>When Clementine starts</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QRadioButton" name="b_always_show_">
        <property name="text">
         <string>Always show the main window</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="b_always_hide_">
        <property name="text">
         <string>Always hide the main window</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="b_remember_">
        <property name="text">
         <string>Remember from last time</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="resume_after_start_">
        <property name="text">
         <string>Resume playback on start</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="stop_play_if_fail_">
        <property name="text">
         <string>Stop playback if song fails to play</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="b_grey_out_deleted_">
        <property name="text">
         <string>Grey out non existent songs in my playlists</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="b_click_edit_inline_">
        <property name="toolTip">
         <string>If activated, clicking a selected song in the playlist view will let you edit the tag value directly</string>
        </property>
        <property name="text">
         <string>Enable song metadata inline edition with click</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="title">
      <string>Using the menu to add a song will...</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_18">
      <item>
       <widget class="QComboBox" name="menu_playmode">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>Never start playing</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Play if there is nothing already playing</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Always start playing</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_previous">
     <property name="title">
      <string>Pressing &quot;Previous&quot; in player will...</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_19">
      <item>
       <widget class="QComboBox" name="menu_previousmode">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>Jump to previous song right away</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Restart song, then jump to previous if pressed again</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="title">
      <string>Double clicking a song will...</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QComboBox" name="doubleclick_addmode">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>Append to the playlist</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Replace the playlist</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string comment="Refers to behavior settings in Clementine settings page.">Open in new playlist</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Add to the queue</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="doubleclick_playmode">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>Never start playing</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Play if there is nothing already playing</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Always start playing</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_doubleclick_playlist">
     <property name="title">
      <string>Double clicking a song in the playlist will...</string>
     </property>
     <layout class="QVBoxLayout" name="layout_doubleclick_playlist">
      <item>
       <widget class="QComboBox" name="doubleclick_playlist_addmode">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>Change the currently playing song</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Add to the queue</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_seekstep">
     <property name="title">
      <string>Seeking using a keyboard shortcut or mouse wheel</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_seekstep">
        <property name="text">
         <string>Time step</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="seek_step_sec">
        <property name="suffix">
         <string> s</string>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_seekstep">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_9">
     <property name="title">
      <string>When saving a playlist, file paths should be</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="b_automatic_path">
        <property name="text">
         <string>Automatic</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="b_absolute_path">
        <property name="text">
         <string>Absolute</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="b_relative_path">
        <property name="text">
         <string>Relative</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="b_ask_path">
        <property name="text">
         <string>Ask when saving</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_7">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="b_write_metadata">
        <property name="text">
         <string>Write metadata</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
